homeassistant/components/goalzero/config_flow.py

```diff
@@ -64,6 +100,21 @@ async def async_step_user(self, user_input=None):
         )
 
     async def _async_try_connect(self, host):
-        session = async_get_clientsession(self.hass)
-        api = Yeti(host, self.hass.loop, session)
-        await api.get_state()
+        """Try connecting to Goal Zero Yeti."""
+        errors = {}
+        try:
+            session = async_get_clientsession(self.hass)
+            api = Yeti(host, self.hass.loop, session)
+            await api.sysinfo()
+        except exceptions.ConnectError:
+            errors["base"] = "cannot_connect"
+            _LOGGER.error("Error connecting to device at %s", host)
+            return errors
+        except exceptions.InvalidHost:
+            errors["base"] = "invalid_host"
+            _LOGGER.error("Invalid host at %s", host)
+            return errors
+        except Exception:  # pylint: disable=broad-except
+            _LOGGER.exception("Unexpected exception")
+            errors["base"] = "unknown"
+            return errors
```

Comment:
```suggestion
            return {"base": "unknown"}
```

Reply:
But that's probably not what you want since you are doing `return self.async_abort(reason=errors)`.

You probably want to return just `unknown`